### ISSUE TITLE
Supported Gigabyte's X570 and Z390 MBs?

### ISSUE THREAD

**Issue by windows2000bug** [jackun/IT8297-rgb-controller#1]
I was looking around and stumbled across this Repo, are you developing an alternative tool to control the MCU on Gigabyte's motherboard?

**Comment by jackun** [comment]
Not really. Just something for myself to maybe use with Prismatik and set leds in linux.
But I have a feeling I'll have to because google is barren of any info about these things and this repo is going to be \#1 search result probably, hah.
